client/tests/webdriver/pages/showTask.page.js

```diff
@@ -52,6 +52,18 @@ class ShowTask extends Page {
     )
   }
 
+  async getNextPeriodButton(assessmentKey, recurrence) {
+    return (await this.getAssessmentResults(assessmentKey, recurrence)).$(
+      "button:last-child"
+    )
+  }
+
+  async getFutureAddAssessmentButton(assessmentKey, recurrence) {
+    return (await this.getAssessmentsTable(assessmentKey, recurrence)).$(
+      "tbody tr:last-child td:last-child"
```

Comment:
This table cell always exists, so this doesn't really test anything. The selector should include the button, so:
`"tbody tr:last-child td:last-child button"`